odoo/addons/base/models/ir_actions.py

```diff
@@ -135,6 +135,34 @@ def get_bindings(self, model_name):
             result['action'] = sorted(result['action'], key=lambda vals: vals.get('sequence', 0))
         return result
 
+    @api.model
+    def _for_xml_id(self, module, xml_id):
```

Comment:
If you change the method name, maybe you could improve the API:
```suggestion
    def _for_xml_id(self, full_xml_id):
```
The separation of the module was coming from the old API of the resolution, which was itself an artefact from the implementation.